test/Microsoft.IdentityModel.Tokens.Saml.Tests/SkipValidationDelegates.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+//TODO: Remove this file and use the one in TestUtils once new validation model is public.
+#nullable enable
+namespace Microsoft.IdentityModel.Tokens.Saml.Tests
+{
+    public static class SkipValidationDelegates
```

Comment:
Can we not use these validators: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/8341a8b493e9a058b358e1c39462ba5fb75dee01/test/Microsoft.IdentityModel.TestUtils/SkipValidationDelegates.cs#L13C4-L13C5

Reply:
Updated, I did not see we had an InternalsVisibleTo.cs class separate from assemblyInfo.cs to manage it on Testutils. Thanks Brent!